### ISSUE TITLE
Latest version is throwing a strange error

### ISSUE THREAD

**Issue by thelinuxlich** [neumino/rethinkdbdash#167]
I changed nothing, only updated rethinkdbdash
```
Unhandled rejection ReqlDriverError: No more rows in the Cursor.
10/27/15 08:02:06
aidax-oracle-1 aidax[12012]: at /app/releases/20151027215847/node_modules/rethinkdbdash/lib/cursor.js:57:14
10/27/15 08:02:06
aidax-oracle-1 aidax[12012]: at Promise._resolveFromExecutor (/app/releases/20151027215847/node_modules/bluebird/js/release/promise.js:457:18)
10/27/15 08:02:06
aidax-oracle-1 aidax[12012]: at Promise._execute (/app/releases/20151027215847/node_modules/bluebird/js/release/debuggability.js:140:9)
10/27/15 08:02:06
aidax-oracle-1 aidax[12012]: at Async._drainQueues (/app/releases/20151027215847/node_modules/bluebird/js/release/async.js:135:10)
10/27/15 08:02:01
aidax-oracle-1 aidax[12012]: at processImmediate [as _immediateCallback] (timers.js:374:17)
10/27/15 08:02:01
aidax-oracle-1 aidax[12012]: at Immediate.Async.drainQueues [as _onImmediate] (/app/releases/20151027215847/node_modules/bluebird/js/release/async.js:16:14)
10/27/15 08:02:01
aidax-oracle-1 aidax[12012]: at Async._drainQueues (/app/releases/20151027215847/node_modules/bluebird/js/release/async.js:135:10)
10/27/15 08:02:01
aidax-oracle-1 aidax[12012]: at Async._drainQueue (/app/releases/20151027215847/node_modules/bluebird/js/release/async.js:125:16)
10/27/15 08:02:01
aidax-oracle-1 aidax[12012]: at Promise._settlePromise0 (/app/releases/20151027215847/node_modules/bluebird/js/release/promise.js:594:10)
10/27/15 08:02:01
aidax-oracle-1 aidax[12012]: at Promise._settlePromises (/app/releases/20151027215847/node_modules/bluebird/js/release/promise.js:677:18)
10/27/15 08:02:01
aidax-oracle-1 aidax[12012]: at Promise._settlePromise (/app/releases/20151027215847/node_modules/bluebird/js/release/promise.js:549:18)
10/27/15 08:02:01
aidax-oracle-1 aidax[12012]: at Promise._settlePromiseFromHandler (/app/releases/20151027215847/node_modules/bluebird/js/release/promise.js:486:31)
10/27/15 08:02:01
aidax-oracle-1 aidax[12012]: at new Promise (/app/releases/20151027215847/node_modules/bluebird/js/release/promise.js:76:14)
10/27/15 08:02:01
aidax-oracle-1 aidax[12012]: at Cursor._next (/app/releases/20151027215847/node_modules/rethinkdbdash/lib/cursor.js:52:11)
10/27/15 08:02:01
aidax-oracle-1 aidax[12012]: at resolve (/app/releases/20151027215847/node_modules/rethinkdbdash/lib/cursor.js:228:14)
10/27/15 08:02:01
aidax-oracle-1 aidax[12012]: at tryCatcher (/app/releases/20151027215847/node_modules/bluebird/js/release/util.js:11:23)
```

**Comment by thelinuxlich** [comment]
Found out the problem, it's because rethinkdbdash is incompatible with bluebird 3.x

**Comment by neumino** [comment]
What the problem with bluebird 3?

**Comment by thelinuxlich** [comment]
It is throwing that error

**Comment by neumino** [comment]
Hum `Promise.reject` doesn't return an operational error anymore...

**Comment by phparkle** [comment]
Getting the same error here

**Comment by neumino** [comment]
I'm traveling for the next few days and have a hard time properly fixing it. I'll merge @thelinuxlich PR to pin bluebird for now and take a look at that this week end.

**Comment by neumino** [comment]
Fixed in 2.1.13 for now by pinning a version of bluebird.
Tracking the migration to bluebird 3 with #170

**Comment by internalfx** [comment]
@neumino Thanks for the quick fix.
